ROADMAP MAR18-SEP18.md

```diff
@@ -13,46 +13,60 @@ Here are my current thoughts on a roadmap for our overall engineering organziati
+* Deprioritized ~~Better broadcasting across platforms - our town halls are great, especially now that they can be broadcast over Youtube live.  We should look into broadcasting these across other platforms as well - i.e. going live on Facebook or Twitter at the same time as we go live on Youtube, perhaps even including Twitch.  We need to look into software that would let us broadcast over multiple platforms - or at the very least push announcements that we are live to multiple platforms.  Eventually I would like to see this show on the main home page of the site, but that is not within this six month roadmap. 
 
 ## Front End
 
-* Policy Page (this is needed for an upcoming event, Jameel Matin will be providing the copy) *COMPLETE*
+* ~~Policy Page (this is needed for an upcoming event, Jameel Matin will be providing the copy)~~ *COMPLETE*
 
-* Templates for static pages.  One of the biggest bottlenecks over the past year has been React - it can be a difficult framework to get started with, especially when someone is brand new to web development.  Rather than attempt to rewrite our front end in a different framework, I’d like to focus on making our particuarly implementation of React more beginner friendly.  Adding in static content is a great way for a beginner to get started with contributing - let’s make a template that can be copy and pasted for any new static page.  Yes, we might violate the DRY rule a bit here, but I think this is a worthy exception and necessary to our mission.  We need the process of using this template very well documented.  Any new copy for the front end should go in a Github issue with a link to this guide, and these issues should be reserved for first time contributions whenever possible.
+* Templates for static pages.  One of the biggest bottlenecks over the past year has been React - it can be a difficult framework to get started with, especially when someone is brand new to web development.  Rather than attempt to rewrite our front end in a different framework, I’d like to focus on making our particuarly implementation of React more beginner friendly.  Adding in static content is a great way for a beginner to get started with contributing - let’s make a template that can be copy and pasted for any new static page.  Yes, we might violate the DRY rule a bit here, but I think this is a worthy exception and necessary to our mission.  We need the process of using this template very well documented.  Any new copy for the front end should go in a Github issue with a link to this guide, and these issues should be reserved for first time contributions whenever possible. _does kyle's work negate this?_
```

Comment:
My work doesn't necessarily negate this, but I do think the new repo will make it easier for people to create pages, with less help, and do them faster

Reply:
@kylemh - how far are we talking about - a CMS? by integrating a CMS, contribution is not necessarily via code. Hoping with this new repository, we can quickly add content. Currently a very long process of making a route, creating container for it. New system will have routes.js and a pages folder, that's it.
@hollomancer - We are inevitably moving towards a CMS so we need to factor that in
@nellshamrell - Need to add a task to assess this for next roadmap. Leaning towards part of it being a CMS, part of it being code-driven.
@kylemh - Contentful? Wordpress?